### PR TITLE
[Cleanup commons-lang] LPD-19260 Remove commons-lang EqualsBuilder and HashCodeBuilder usage

### PR DESCRIPTION
**Pull request by julschong**

### CODE REVIEW

**Comment by julschong** [comment]
ci:test:relevant

File: portal-impl/src/com/liferay/portal/plugin/PluginPackageImpl.java [hunk 1]
```diff
@@ -166,11 +164,7 @@ public String getVersion() {
 
 	@Override
 	public int hashCode() {
-		HashCodeBuilder hashCodeBuilder = new HashCodeBuilder();
-
-		hashCodeBuilder.append(getModuleId());
-
-		return hashCodeBuilder.hashCode();
+		return getModuleId().hashCode();
```
Comment (tinatian) [inline portal-impl/src/com/liferay/portal/plugin/PluginPackageImpl.java:167]: @julschong Method chain here is not allowed, please do:
String moduleId = getModuleId();
return moduleId.hashCode();

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
avoid method chaining

**Comment by julschong** [comment]
ci:report:2545793337

**Comment by julschong** [comment]
no unique failure

**Comment by tinatian** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/147806

**Pull request closed without merging**